src/applications/claims-status/tests/components/ClaimContentionList.unit.spec.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+import ClaimContentionList, {
+  MAX_CONTENTIONS,
+} from '../../components/ClaimContentionList';
+
+describe('ClaimContentionList', () => {
+  it('renders the first MAX_CONTENTIONS amount of contentions by default', () => {
+    const contentions = [];
+    for (let i = 0; i < MAX_CONTENTIONS + 1; i += 1) {
+      contentions.push({ name: `Contention ${i}` });
+    }
+    const { queryByText } = render(
+      <ClaimContentionList contentions={contentions} />,
+    );
+
+    expect(queryByText(contentions[MAX_CONTENTIONS - 1].name)).to.exist;
+    expect(queryByText(contentions[MAX_CONTENTIONS].name)).to.not.exist;
+  });
+
+  it('should render a button to show all contentions if there are more than MAX_CONTENTIONS', () => {
+    const contentions = [];
```

Comment:
I would just create an array of contentions and slice it instead of generating new ones for each tests

Reply:
I moved the mock contentions generation into a reusable scope so it would always work for MAX_CONTENTIONS. Slice wasn't needed in this case.